### ISSUE TITLE
big_time_between showing 1 decimal place for hour field

### ISSUE THREAD

**Issue by WhimsySpoon** [Petro31/easy-time-jinja#25]
Since installing v2, the hour part of big_time_between is rounded, but given a `.0` decimal place.

For example, the following returns `1.0 hour and 59 minutes`:

````
{% from 'easy_time.jinja' import big_time_between %}
{{ big_time_between(now(), now() + timedelta( hours = 1, minutes = 59 )) }}
````

**Comment by Petro31** [comment]
Rounding does not happen in big time.  Rounding only occurs in easy time or if you have a single value being returned, like `hours` or `minutes`.  Not `hours` and `minutes`.

**Comment by aaronspittlespirum** [comment]
Thanks, and apologies for the lack of clarity on my initial post.

To confirm, the bug isn't around rounding, it's that since v2, a `.0` is being appended to the hour value.

**Comment by Petro31** [comment]
Fixed in 2.0.0.1
